Automattic/wp-calypso
# Domains: Add copy to map or transfer domain in the removal dialog

#### Changes proposed in this Pull Request

This PR updates the domain removal dialog, to include a copy that lets user know they can map their domain to another site, or they can also transfer it to another provider.


#### Testing instructions

To test this, you need either a domain purchased with credits or a domain outside the refund window.

- Go to Payment Settings for a domain you own
- Click on the Remove domain button and verify by checking against the screenshots below
- Verify that the links in the dialog take you to the correct support page

You should see this dialog (pay attention to both the message and the buttons):

<img width="1138" alt="Screenshot 2020-01-10 at 1 43 33 PM" src="https://user-images.githubusercontent.com/13062352/72153817-346be100-33af-11ea-8af0-17c911514ff3.png">

NOTE: if the domain is within refund period, you might see an additional piece of text in the dialog below the changes introduced here. That is expected.
